maintainers/maintainer-list.nix

```diff
@@ -2565,6 +2565,12 @@
     githubId = 119483;
     name = "Matthew Brown";
   };
+  eduardosm = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add eduardosm``.

Reply:
Done (https://github.com/NixOS/nixpkgs/pull/108041/commits/88b72c33d73a5fdc176cec6b50b3034fc3e7a4f2)